seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
```

Comment:
**Security Issue**: The conversion from byte array to Integer using Bytes.toString(cell) can lead to NumberFormatException if the byte array does not contain a valid integer representation. This is a critical input validation issue. <br> **Fix**: Validate the byte array content before conversion to ensure it contains a valid integer representation. <br> **Code Suggestion**: 
 ```
 - return Integer.valueOf(Bytes.toString(cell));
 + if (cell != null && org.apache.commons.lang3.StringUtils.isNumeric(Bytes.toString(cell))) {
 +     return Integer.valueOf(Bytes.toString(cell));
 + } else {
 +     throw new IllegalArgumentException("Invalid byte array for integer conversion");
 + }
 ```